README.md

```diff
@@ -24,6 +68,50 @@ The `pages/api` directory is mapped to `/api/*`. Files in this directory are tre
 
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
 
+## Deploy on Vercel
+
+The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+
+Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+
+
+## Usage
+
+- Home Page-
```

Comment:
Bas ek home page ka screenshot daal do baaki pages rehne do uski jagah baaki pages ke bare me 2-4 line description de do and also mention to give feedback in contact us page and mention functionality of join us.